### DISCUSSION TITLE
能否增加图片预览的翻页功能

### DISCUSSION THREAD

**Discussion by funny1573** [AlistGo/alist#2695] [Ideas]
作者你好,请问能否在图片预览界面增加"上一张""下一张"的按钮,以便快速预览连续的图片
![N4C}IYT2JZES9Y_@4{F%58H](https://user-images.githubusercontent.com/96622280/207211868-f54c784f-ef1e-4d46-83b2-250dc5e91133.png)
如百度网盘的效果

**Comment by anwen-anyi** [comment]
右上角自行切换模式
